ios/PiwikProSdk.m

```diff
@@ -92,4 +111,14 @@ @implementation PiwikProSdk
   }
 }
 
+- (NSDictionary *)constantsToExport
```

Comment:
Seems unnecessary to move the values back and forth over the react native bridge. Maybe better to just pass a string value to the `setCustomDimension` method and handle the conversion on the native side.